config/prod/elevatems_features.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v8.j2
```

Comment:
this is for aws linux distro.  you'll want to use the ubuntu template: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/147/files

Reply:
I changed the template to the Ubuntu one